mycore-viewer/src/main/typescript/modules/pdf/components/MyCoRePDFViewerComponent.ts

```diff
@@ -66,11 +67,11 @@ namespace mycore.viewer.components {
             if (this._settings.doctype == "pdf") {
                 this._pdfUrl = ViewerFormatString(this._settings.pdfProviderURL, { filePath: this._settings.filePath, derivate: this._settings.derivate });
                 var workerURL = this._settings.pdfWorkerURL;
-                PDFJS.workerSrc = workerURL;
+                pdfjsLib.workerSrc = workerURL;
                 var that = this;
                 var pdfLocation = this._pdfUrl;
-                PDFJS.getDocument(pdfLocation).then((pdfDoc) => {
-                    this._pdfDocument = <PDFDocumentProxy>pdfDoc;
+                (<any> pdfjsLib.getDocument(pdfLocation)).promise.then((pdfDoc:PDFDocumentProxy) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['pdfjsLib' is not defined.](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=5286493)